Vagrantfile

```diff
@@ -0,0 +1,44 @@
+# -*- mode: ruby -*-
```

Comment:
https://www.vagrantup.com/  it's an easy way to make reproduceable lightweight VMs.  In this case it provisions a server based on ubuntu 14.04.  Then it runs the vagrant-dependencies script on it which sets up the environment.

If you install the latest version of vagrant, and in the directory with the Vagrantfile, you type: ```vagrant up``` then it will download and install this machine.  It will also start it running.  ```vagrant ssh``` will connect you to that machine, then you can do cd chesster and ./start_debug and it should run on that machine.  ```vagrant halt``` will bring the machine down. One minor annoying thing about vagrant is that the machines stay running in the background until you stop them. :/

Eithe way, I should clean this up - it still refers to "modster". It should refer to chesster.  I'll do that now.